components/expenses/Expense.js

```diff
@@ -438,15 +445,16 @@ class Expense extends React.Component {
                 <Span color="red.500">{intl.formatMessage(this.messages['expenseTypeMissing'])}</Span>
               )}
               {mode !== 'edit' && (canPay || canApprove || canReject || canMarkExpenseAsUnpaid) && (
-                <div className="manageExpense">
-                  {canPay && expense.payoutMethod === 'other' && (
+                <Flex flexDirection="column">
+                  {canPay && (
                     <EditPayExpenseFeesForm
                       canEditPlatformFee={LoggedInUser.isRoot()}
                       currency={collective.currency}
                       onChange={fees => this.handleChange({ fees })}
+                      payoutMethod={expense.payoutMethod}
                     />
                   )}
-                  <div className="expenseActions" data-cy="expense-actions">
+                  <ExpenseActionsWrapper className="expenseActions" data-cy="expense-actions">
```

Comment:
Not a requested change: `ExpenseActionsWrapper` doens't seems necessary, you can achieve the same effect by using `Flex`. Also isn't that an opportunity to get rid of the `expenseActions` class?